### PR TITLE
German translation for http proxies

### PR DESCRIPTION
**Pull request by boris779**
### Description
German translation for http proxies
Also added a missing : in the english version and non translated pages

### Motivation and Context
Missing german translation for that page

### Types of changes
- [ ] Change to the site (I am attaching a screenshot showing the before and after)
- [ ] Code example added (and I also added the example to all translated languages)
- [x] Improved translation
- [x] Added new translation (and I also added a notice to each document missing translation)

### Checklist
- [X] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: docs_source_files/content/webdriver/http_proxies.de.md [hunk 1]
```diff
@@ -1,23 +1,25 @@
...
+Ein Proxy-Server fungiert als Zwischenstation für Anfragen zwischen
+Client und Server. Simpel erklärt werden die Daten über den Proxy zu der
+gewünschten Internetadresse und wieder retour geleitet.
 
-A proxy server for automation scripts with Selenium could be helpful for:
-- Capture network traffic
-- Mock backend calls made by the website
-- Access the required website under complex network topologies or strict
-corporate restrictions/policies.
+Ein Proxy Server für die automatisierten Seleniumskripte kann hilfreich
```
Comment (harsha509) [inline docs_source_files/content/webdriver/http_proxies.de.md]: can we add any space or - for 'Seleniumskripte'. 
Its ok if Seleniumskripte is correct. can you please suggest on this @boris779 !

**Review by harsha509 — CHANGES_REQUESTED** [review]
Thank you @boris779 !

Can you look into the comment once.

File: docs_source_files/content/webdriver/http_proxies.de.md (hunk 1 above)
Comment (boris779) [reply docs_source_files/content/webdriver/http_proxies.de.md]: I was also unsure about this word. I changed it to Selenium-Skripte, even if Words with dash are not common in German.

**Review by harsha509 — APPROVED** [review]
Thank you @boris779 !

**Pull request merged**